HRIS.Services/Services/ErrorLoggingService.cs

```diff
@@ -56,7 +56,7 @@ public Exception LogException(Exception exception)
             {
                 DateOfIncident = targetLocalTime,
                 Message = exception.Message,
-                StackTrace = exception.StackTrace,
+                StackTrace = exception.ToString(),
                 IpAddress = "",
                 RequestBody = "",
                 RequestUrl = "",
```

Comment:
Refactor: not sure on the standard for this, "" versus string.Empty.

Reply:
Personally I prefer `string.Empty` but they both achieve the same result.